virttest/test_setup.py

```diff
@@ -1438,6 +1438,17 @@ def check_vfs_count(self):
         expected_count = int((re.findall("(\d+)", self.driver_option)[0])) * len(self.get_pf_ids())
         return (self.get_vfs_count() == expected_count)
 
+    def get_controller_type(self):
+        """
+        Get the Controller Type for SR-IOV Mellanox Adapter PFs
```

Comment:
I dont think the check is specific to mellanox here.. with given command and pf_filter we can determine any other vendor card type on power.. please double check and update

Reply:
right!

Reply:
@bssrikanth @balamuruhans I agree , this isn't specific to mellanox. Have made the changes, please review.